doc/help/elektra-highlevel-gen.md

```diff
@@ -0,0 +1,410 @@
+# elektra-highlevel-gen(7) -- High-level API code-generation advanced features
+
+This document focuses on the advanced features of the high-level API code-generator template. We assume you already familiarized yourself
+with the basic features explained in [`kdb-gen-highlevel(1)`](kdb-gen-highlevel.md).
+
+## Configuration Options
+
+The parameters that are relevant to the concepts described here are (for the rest see [`kdb-gen-highlevel(1)](kdb-gen-highlevel.md)):
+
+- `enumConv`: allowed values: `strcmp`, `switch`, `default` (default)
```

Comment:
"auto" instead of "default" might be less confusing.

And reorder these items here so that they match your later explanations.

Reply:
> And reorder these items here so that they match your later explanations.

Reorder in what way? The order matches exactly with the following paragraphs.

Reply:
Forward references like "The `enumConv` option is described [below](#enums)." should be avoided. Moving enumConv the last of these 3 entries would solve that problem (the sentence could be removed).